tide_landing_page.install

```diff
@@ -2438,3 +2438,23 @@ function tide_landing_page_update_8043() {
     $config->set('content.field_statistic_block.settings.default_paragraph_count', '2')->save();
   }
 }
+
+/**
+ * Update and Add new options for "Show table of contents" "Display heading".
+ */
+function tide_landing_page_update_8044() {
```

Comment:
hey @mayurngondhkar 
why the update hook resides in `tide_landing_page` module? should it be part of https://github.com/dpc-sdp/tide_core/pull/312/

Reply:
Hey @vincent-gao ,

Yup,  I've moved it to tide_core 👍